common/src/main/java/whocraft/tardis_refined/common/util/TardisHelper.java

```diff
@@ -44,8 +44,8 @@ public static void playCloisterBell(TardisLevelOperator tardisLevelOperator) {
 
     public static boolean isInArsArea(BlockPos blockPos) {
 
-        BlockPos corner1 = new BlockPos(1009, 97, -2);
-        BlockPos corner2 = new BlockPos(1041, 118, 30);
+        BlockPos corner1 = new BlockPos(1051, 97, 6);
```

Comment:
If this never changes, it should be a static field

Reply:
Moved to constants